### PR TITLE
KEYCLOAK-11680 logo click also returns to referrer

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: themes/src/main/resources/theme/keycloak-preview/account/resources/app/App.tsx [hunk 1]
```diff
@@ -65,7 +66,7 @@ export class App extends React.Component<AppProps> {
 
         const Header = (
             <PageHeader
-                logo={<Brand src={brandImg} alt="Logo" className="brand"/>}
+                logo={<ReferrerLink><Brand src={brandImg} alt="Logo" className="brand"/></ReferrerLink>}
```
Comment (ssilvert) [inline themes/src/main/resources/theme/keycloak-preview/account/resources/app/App.tsx:69]: This won't work if there is no referrer.

**Review by ssilvert — COMMENTED** [review]
I'm not sure what the URL should be here, but it can't be the referrer because that doesn't make sense for the logo and referrer might not exist.

@vmuzikar suggested that it go back to the Welcome page.  I guess that is as good as anything at the moment.  But I really think we should just hold off on this until we have a fix for https://issues.redhat.com/browse/KEYCLOAK-12776.  Both the image and the url for the image should be configurable.

**Comment by vmuzikar** [comment]
@ssilvert I don't think this is blocked by KEYCLOAK-12776 as you're mentioning. The image src is not affected by the link URL and the URL will not be configurable, am I correct?
That said, I'm ok either way – this can wait for sure.

**Comment by edewit** [comment]
Going back to the welcome screen doesn't make sense as it's only a temporary page there is nothing there that you might need once react is started.

**Comment by vmuzikar** [comment]
@edewit I'd say Welcome Screen is a good dashboard. Btw, Google does it that way too – you can click the header and it brings you back to the "Welcome Screen". ;)

**Comment by edewit** [comment]
It's all fine with me, but seems that @ssilvert wants something more fancy then just a link (no clue what that should be), so closing this.

**Pull request closed without merging**